src/goto-programs/adjust_float_expressions.cpp

```diff
@@ -33,11 +31,11 @@ static bool have_to_adjust_float_expressions(
      expr.id()==ID_floatbv_typecast)
     return false;
 
-  const typet &type=ns.follow(expr.type());
+  const typet &type = expr.type();
```

Comment:
Why is this follow not necessary?

Reply:
This hasn't been necessary for many years (ever since typedefs get expanded immediately).